sandbox/grommet-app/src/App.jsx

```diff
@@ -80,30 +76,23 @@ const App = () => {
         ) : (
           <BackgroundContext.Provider value={contextValue}>
             <WorkspaceContext.Provider value={workspaceContextValue}>
-              <GlobalHeader
-                darkMode={darkMode}
-                setDarkMode={setDarkMode}
-                setActiveTheme={setActiveTheme}
-                activeTheme={activeTheme}
-                backgroundBack={backgroundBack}
-                setBackgroundBack={setBackgroundBack}
-                workspace={workspace}
-                setWorkspace={setWorkspace}
-              />
               <BrowserRouter>
+                <GlobalHeader
+                  darkMode={darkMode}
+                  setDarkMode={setDarkMode}
+                  setActiveTheme={setActiveTheme}
+                  activeTheme={activeTheme}
+                  backgroundBack={backgroundBack}
+                  setBackgroundBack={setBackgroundBack}
+                  workspace={workspace}
+                  setWorkspace={setWorkspace}
+                />
                 <Routes>
-                  <Route
-                    path="/"
-                    element={
-                      workspace === 'Acme Production' ? (
-                        <Home />
-                      ) : (
-                        <NextDashboard />
-                      )
-                    }
-                  />
+                  <Route path="/" element={<Home />} />
                   <Route path="/sustainability" element={<Sustainability />} />
                   <Route path="/sticker-sheet" element={<StickerSheet />} />
+                  {/* TO DO unfinished layout */}
+                  {/* <Route path="/refresh" element={<Refresh />} /> */}
```

Comment:
How is this different than line 96 immediately following?

Reply:
This was missed when I did the merge. We can remove the "/refresh" route (the layout on that page is already in the spirit of your latest "Collections" page layout. Will update now to remove.